cmd/vpu_plugin/vpu_plugin_test.go

```diff
@@ -62,6 +103,69 @@ func (n *fakeNotifier) Notify(newDeviceTree dpapi.DeviceTree) {
 	n.scanDone <- true
 }
 
+func TestScanPci(t *testing.T) {
+	var fN fakeNotifier
+
+	f, err := os.Create(hddlSocketPci)
+	if err != nil {
+		t.Error("create fake hddl file failed")
+	}
+
+	//create a temporary folder to create fake devices files for PCI scanning
+	tmpPciDir, err := os.MkdirTemp("/tmp", "fake-pci-devices")
```

Comment:
this is available only in Go v1.16 which we don't have yet so the CI test fails.

Reply:
Any suggestion what we should update?

Reply:
#630 has examples for the "inverse":  `ioutil.TempDir`

Reply:
Shall I wait for your update be merged first?

Reply:
let's not wait that but use `ioutil.TempDir` in this PR.